lua/init.lua

```diff
@@ -78,8 +78,19 @@ for _,global_variable_setting in ipairs(args.global_variable_settings) do
 end
 
 for _,option_setting in ipairs(args.option_settings) do
+    -- Ignore initial values of lines and columns because they are set by neovim directly.
+    -- See https://github.com/neovide/neovide/issues/2300
+    if option_setting ~= "lines" and option_setting ~= "columns" then
+        vim.api.nvim_create_autocmd({ "VimEnter" }, {
+            pattern = option_setting,
+            once = true,
+            nested = true,
+            callback = function()
+                rpcnotify("option_changed", option_setting, vim.o[option_setting])
+            end
+        })
+    end
     vim.api.nvim_create_autocmd({ "OptionSet" }, {
-        pattern = option_setting,
```

Comment:
Doesn't the removal of this cause it to be run for all option changes now.? And since it's done in a loop many times per option.

Reply:
Mmmm yes. Obvious good point.